components/user-store/org.wso2.carbon.identity.user.store.configuration.ui/src/main/resources/web/userstore_config/userstore-config.jsp

```diff
@@ -806,7 +828,7 @@
             //no advanced properties
         }
     %>
-    <input type="hidden" name="defaultProperties" id="defaultProperties" value=<%=i%>/>
+    <input type="hidden" name="defaultProperties" id="defaultProperties" value=<%=Encode.forHtmlAttribute(i)%>/>
```

Comment:
Exception is thrown since the integer is not casted to a String. Method accepts only string arguments. This encoding should be removed ideally